src/transformState.js

```diff
@@ -5,7 +5,47 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {
+    const whatWeMustDo = action.type;
+    const isAdding = whatWeMustDo === 'addProperties';
+    const isDeleting = whatWeMustDo === 'removeProperties';
+    const isClearing = whatWeMustDo === 'clear';
+
+    switch (true) {
```

Comment:
why do you use `switch (true)` when you can switch by `action.type`? please, redo it